serial-prober.js

```diff
@@ -174,6 +174,10 @@ class SerialProber {
     if (port.path.startsWith('/dev/tty.usb')) {
       port.path = port.path.replace('/dev/tty', '/dev/cu');
     }
+    if (this.param.allowRPIAMASerial && port.path.startsWith('/dev/ttyAMA')) {
```

Comment:
Please rename `allowRPIAMASerial` to `allowAMASerial` and also update the README.